board_common/native/i2c_eusci_native.h

```diff
@@ -0,0 +1,72 @@
+#ifndef _BOARD_COMMON_NATIVE_I2C_H_
+#define _BOARD_COMMON_NATIVE_I2C_H_
+
+/// XXX: TODO: this should probably be exported to a common header
+#include <msp430.h>
+#if defined ( MSP430_CLASS_FR5xx_6xx )
+#   include "msp430fr5xx_6xxgeneric.h"
+#else
+#   error "Unspported processor"
+#endif
+
+#include <driverlib.h>
+
+#ifndef __MSP430_HAS_EUSCI_Bx__
+#   error "We need EUSCI for I2C"
+#endif
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * Represents which EUSCI module we're hooked up to
+ */
+typedef enum eusci {
+#ifdef EUSCI_A0_BASE
+    EUSCI_A0,
+#endif
+#ifdef EUSCI_A1_BASE
+    EUSCI_A1,
+#endif
+#ifdef EUSCI_A2_BASE
+    EUSCI_A2,
+#endif
+#ifdef EUSCI_A3_BASE
+    EUSCI_A3,
+#endif
+#ifdef EUSCI_B0_BASE
+    EUSCI_B0,
+#endif
+#ifdef EUSCI_B1_BASE
+    EUSCI_B1,
+#endif
+#ifdef EUSCI_B2_BASE
+    EUSCI_B2,
+#endif
+#ifdef EUSCI_B3_BASE
+    EUSCI_B3,
+#endif
+    EUSCI_count
+} eusci_t;
```

Comment:
I think the time to resolve that TODO is probably now, either in this branch or in @branw's SPI branch

Reply:
Fixed in 40d74ff2bbfdeb6b6cd29f630048d17bc47b1fa0 for #22